Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3186,6 +3186,19 @@ InlinePredictionsEnabled:
     WebCore:
       default: false
 
+InlinePredictionsInAllEditableElementsEnabled:
+  type: bool
+  status: internal
+  humanReadableName: "Inline Text Predictions"
+  humanReadableDescription: "Enable Inline Text Predictions in all contenteditable elements"
```

Comment:
Would rewrite this and the PR title to refer to "editable elements" rather than "contenteditable elements", since it also covers `<input>` and `<textarea>`.

Reply:
Is it though? Because this check is also present
```
    if (!editorState.isContentEditable)
        return std::nullopt;
```

Reply:
It does, since those elements have a `contenteditable` in their shadow DOM.

Though that's an implementation detail, whereas this flag is driving "user"-facing UI.

Reply:
ohhh, forgot they did that. Will change the  description / PR name